### ISSUE TITLE
It seems like disabling the loading of the next video is not working anymore

### ISSUE THREAD

**Issue by mzso** [YePpHa/maia-yt#131]
Hi!

Nothing else to add. In the past few days this annoying feature is activated despite project-maia.

**Comment by JayBrown** [comment]
You're right. It seems YouTube themselves changed something. In my case the site reverted back from dark to light mode, and autoplay-next was reenabled. I just disabled it again, but not with maia, so it's possible that the maia setting (prevent autoplay) currently isn't working.
